### ISSUE TITLE
Querying Info on a download resets Installed/uninstalled state

### ISSUE THREAD

**Issue by Al12rs** [ModOrganizer2/modorganizer#1452]
Each download has a single state, either downloading, querying, pausing, paused, ready (downloaded), installed, uninstalled etc. 

When an installed/uninstalled download is queried the state is changed to querying/queryingMd5 and when it is finished it is set to Ready (downloaded) instead of being reverted to whatever it was before.

Reproduced on 2.4.0 but it should be a problem with earlier versions as well.

**Comment by Ranvir1995** [comment]
Yes ive got the same issue as seen in the screenshot below...only manual download works. pls help g
![Screenshot (9)](https://user-images.githubusercontent.com/80562776/111019267-18653480-83e4-11eb-9e39-47ab75531b9b.png)
uys

**Comment by isanae** [comment]
@Ranvir1995 Your problem has nothing to do with this. Do not post the same question to multiple, random issues.
